### ISSUE TITLE
TypeScript Install Error

### ISSUE THREAD

**Issue by philcockfield** [philcockfield/ui-harness#74]
Create a new module:

      npm init -f

Install the UIHarness:

    npm i -S ui-harness@3.11.0-beta1

Error occurs in `postinstall` step:

![image](https://cloud.githubusercontent.com/assets/185555/16169678/f9b7c982-358a-11e6-962e-5938bd3c29c6.png)

    > ui-harness@3.11.0-beta1 postinstall /Users/phil/Dropbox (Personal)/TEMP/foo/node_modules/ui-harness
    > typings install

    sh: typings: command not found
    foo@1.0.0 /Users/phil/Dropbox (Personal)/TEMP/foo
    └── (empty)

**Comment by philcockfield** [comment]
https://github.com/philcockfield/ui-harness/commit/fe74e426a9b9a51b412c3d4860aa70ed4dad635c

Temporarily disable `post install` step so I can make another release.  Prob a smarter way to handle this.

![image](https://cloud.githubusercontent.com/assets/185555/16169837/13169d8a-3591-11e6-95fe-bcc5af24e517.png)

**Comment by frederickfogerty** [comment]
To be honest, the `postinstall` hook was something I just assumed I would work... clearly not. 

I've been doing some research and it turns out that we don't need this. In typings, there are two types of dependencies: external and global (used to be called ambient). If you define external dependencies (which React should be), it will install them recursively. If you define global dependencies (which React is, since we are importing it from DefinitelyTyped (read [here](https://github.com/typings/typings/blob/master/docs/external-modules.md) for more).

TL;DR: we can just remove this line.
